.github/workflows/macos.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: macos-latest
     strategy:
       matrix:
-        go: [1.14, 1.15]
+        go: [1.15, 1.16]
```

Comment:
I'd like to keep 1.14 here because

> As announced in the Go 1.14 release notes, Go 1.15 drops support for 32-bit binaries on macOS, iOS, iPadOS, watchOS, and tvOS (the darwin/386 and darwin/arm ports).

And there are still users who actually need 32-bit binaries on macOS (#2328)

So please change this to `go: [1.14, 1.16]`